hw/ip/adc_ctrl/doc/_index.md

```diff
@@ -1,34 +1,215 @@
 ---
-title: "ADC_CTRL HWIP Technical Specification"
+title: "Analog to Digital Converter Control Interface"
 ---
 
 # Overview
 
-
+This document specifies the ADC controller IP functionality.
+This IP block implements control and filter logic for an analog block that implements a dual ADC.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}})
+See that document for integration overview within the broader top level system
```

Comment:
Nit: missing period at the end of the sentence.

Reply:
Sure.